src/sources.py

```diff
@@ -284,19 +284,54 @@ def parse_obj_arr_obj(obj, c_file, prefix, obj_typename):
         condition = "\n                && ".join( \
             ['strcmp (tree->u.object.keys[i], "%s")' % i.origname for i in obj.children])
         c_file.write("""
-    if (tree->type == yajl_t_object && (ctx->options & OPT_PARSE_STRICT))
-      {
+    if (tree->type == yajl_t_object) {
```

Comment:
in the C code, could you please keep the existing indentation?

The '{' should be on a new line.  Same comments for other occurrences.

Reply:
fixed